sdk/python/tests/test_materialize.py

```diff
@@ -83,20 +83,20 @@ def prep_bq_fs_and_fv(
         event_timestamp_column="ts",
         created_timestamp_column="created_ts",
         date_partition_column="",
-        field_mapping={"ts_1": "ts", "id": "driver_ident"},
+        field_mapping={"ts_1": "ts", "id": "driver_id"},
     )
 
     fv = get_feature_view(bigquery_source)
     e = Entity(
         name="driver_id",
         description="id for driver",
-        join_key="driver_ident",
```

Comment:
this tests that things still work when the join_key is different from the name of the entity (although it should probably be in its own separate test) so can we keep it in?